src/api.js

```diff
@@ -254,6 +254,10 @@
     connect.core.getUpstream().onUpstream(connect.AgentEvents.MUTE_TOGGLE, f);
   };
 
+  Agent.prototype.onLocalMediaStreamCreated = function (f) {
```

Comment:
We are missing documentation for this still

Reply:
Removing this for this release.